src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Debugging/CSharpProximityExpressionResolver.cs

```diff
@@ -2,12 +2,13 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using System.Threading;
 using Microsoft.CodeAnalysis;
 
 namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Debugging
 {
     internal abstract class CSharpProximityExpressionResolver
     {
-        public abstract IReadOnlyList<string> GetProximityExpressions(SyntaxTree syntaxTree, int absoluteIndex);
+        public abstract IReadOnlyList<string> GetProximityExpressions(SyntaxTree syntaxTree, int absoluteIndex, CancellationToken cancellationToken);
```

Comment:
The Roslyn API updated, so I figured we may as well too since the only call-site already has a `CancellationToken`.